lbry/extras/daemon/daemon.py

```diff
@@ -2479,6 +2479,10 @@ async def jsonrpc_claim_search(self, **kwargs):
         Returns: {Paginated[Output]}
         """
         wallet = self.wallet_manager.get_wallet_or_default(kwargs.pop('wallet_id', None))
+        if ("claim_ids" in kwargs and not kwargs["claim_ids"]
+                and "claim_id" in kwargs and kwargs["claim_id"]):
+            # Discard `claim_ids` when it is empty, otherwise `ValueError`.
+            kwargs.pop("claim_ids")
```

Comment:
This can be simplified, don't need the second check and don't need a redundant comment (it's obvious below what's happening: an empty value is popped):
```
        if "claim_ids" in kwargs and not kwargs["claim_ids"]:
            kwargs.pop("claim_ids")
```

Reply:
Well, I only want to discard `'claim_ids'` if `'claim_id'` is present, as if the latter is not present, there is no need to discard it, even if empty, hence the second test.